ui/pages/swaps/view-on-block-explorer/__snapshots__/view-on-block-explorer.test.js.snap

```diff
@@ -3,7 +3,7 @@
 exports[`ViewOnBlockExplorer renders the component with initial props 1`] = `
 <div>
   <div
-    class="box view-on-block-explorer box--margin-top-6 box--flex-direction-row"
+    class="box view-on-block-explorer box--margin-top-6 box--flex-direction-row box--color-text-default box--color-text-default"
   >
     <button>
       View Swap at etherscan.io
```

Comment:
Hmm why is this getting added twice?

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/15565/commits/bcef9b3e149b6dbd993e1f4e55b1516d09edd289